### PR TITLE
Calendar button event detail

### PR DESCRIPTION
**Pull request by MohamedOmarii**
related to #44

### CODE REVIEW

**Comment by JWLD** [comment]
Not really sure this is the best way to do this - another option could be adding a `calendar` property to the view context when you do `res.render`, which is set to either true or false, and wrap the calendar element in an if statement

**Comment by Karyum** [comment]
@JWLD :shipit:

File: views/event_details.hbs [hunk 1]
```diff
@@ -5,7 +5,7 @@
   </div>
 </div>
 
-<div class="event-details text-center">
+<div id="event-details" class="event-details text-center">
```
Comment (JWLD) [inline views/event_details.hbs]: what's this for?

Comment (Karyum) [reply views/event_details.hbs]: yeah not sure when was this added but removed it cheers!

**Pull request merged**